### PR TITLE
Allow testsuite/integration/basic execution with the ts.ee9 profile (round2)

### PR DESCRIPTION
**Pull request by ropalka**

### CODE REVIEW

File: testsuite/integration/basic/pom.xml [hunk 1]
```diff
@@ -3779,8 +3772,9 @@
                                         <exclude>org/jboss/as/test/integration/jsf/version/JSFDeploymentProcessorTestCase.java</exclude>
                                         <exclude>org/jboss/as/test/integration/management/deploy/runtime/JaxrsRuntimeNameTestCase.java</exclude><!-- test expects javax strings in mgmt api responses -->
                                         <exclude>org/jboss/as/test/integration/weld/ejb/SessionObjectReferenceTestCase.java</exclude>
-                                        <exclude>org/wildfly/test/integration/microprofile/jwt/ejb/JWTEJBTestCase.java</exclude><!-- No JWT. TODO why would JWT work in the normal profile? -->
                                         <exclude>org/wildfly/test/integration/web/annotationsmodule/EarModuleDeploymentTestCase.java</exclude>
+                                        <exclude>org/wildfly/test/integration/observability/opentelemetry/RestClientTestCase</exclude>
+                                        <exclude>org/wildfly/test/integration/observability/opentelemetry/BasicOpenTelemetryTestCase</exclude>
```
Comment (bstansberry) [inline testsuite/integration/basic/pom.xml:3777]: Please remove these; https://github.com/wildfly/wildfly/pull/14674 should fix this problem.

**Review by bstansberry — APPROVED** [review]

File: testsuite/integration/basic/pom.xml (hunk 1 above)
Comment (ropalka) [reply testsuite/integration/basic/pom.xml]: Done @bstansberry

**Comment by bstansberry** [comment]
@ropalka FYI I filed https://issues.redhat.com/browse/WFLY-15309 as a general tracker for general "test was excluded in ts.ee9 but now can execute, so unexclude it" work between now and WF 25 Final. I changed the title of this PR to reference that. No need for you to do anything.

**Comment by bstansberry** [comment]
Thanks, @ropalka

**Pull request merged**